packages/eth-deposit/scripts/exec-viaDApp.js

```diff
@@ -66,50 +66,31 @@ const main = async () => {
     BigNumber.from(10000000000000),
     { value: ethToL2DepositAmount }
   )
-  const rec = await depositTx.wait()
-  console.warn('deposit L1 receipt is:', rec.transactionHash)
-
-  /**
-   * With the transaction confirmed on L1, we now wait and check for the L2 side (i.e., balance credited to L2) to be confirmed as well.
-   * First, we get our txn's sequence number from the event logs (using a handy utility method)
-   * This sequence number uniquely identifies our L1-to-L2 message:
-   */
-  const seqNumArr = await bridge.getInboxSeqNumFromContractTransaction(rec)
-  if (seqNumArr === undefined) {
-    throw new Error('no seq num')
-  }
-  console.log('inbox sequence number is found!')
-  /**
-   * Note that a single txn could (in theory) trigger many l1-to-l2 messages; we know ours only triggered 1 tho.
-   */
-  const seqNum = seqNumArr[0]
 
+  const depositRec = await depositTx.wait()
+  console.warn('deposit L1 receipt is:', depositRec.transactionHash)
+  
   /**
-   * Using the sequence number, we can deterministically predict what its corresponding L2 txn will be
+   * Now we wait for L1 and L2 side of transactions to be confirmed
    */
-  const l2TxHash = await bridge.calculateL2TransactionHash(seqNum)
-  console.log('l2TxHash is: ' + l2TxHash)
-
-  /**
-   * ... and now we wait. Here we're waiting for the Sequencer to include the L2 message in its off-chain queue. The Sequencer should include it in under 10 minutes.
-   */
-  console.log('waiting for l2 transaction:')
-  const l2TxnRec = await l2Provider.waitForTransaction(
-    l2TxHash,
-    undefined,
-    1000 * 60 * 12
+  console.log(
+    `Deposit initiated: waiting for L2 retryable (takes < 10 minutes; current time: ${new Date().toTimeString()}) `
   )
-  console.log(`L2 transaction found! ${l2TxnRec.transactionHash}`)
 
   /**
-   * Our L2 balance should now be updated!
+   * We check if l2Wallet ETH balance is properly updated after the deposit
    */
-
-  const l2WalletUpdatedEthBalance = await bridge.getL2EthBalance()
-
-  console.log(
-    `your L2 balance is updated from ${l2WalletInitialEthBalance.toString()} to ${l2WalletUpdatedEthBalance.toString()}`
-  )
+  for (let i = 0; i < 60; i++) {
```

Comment:
I think we should at least add a comment here explaining that we're just manually waiting for the eth balance to update to illustrate what's going on, but to do this programatically / the right way, see `exec-viaLib`. Basically we don't anyone to check for eth deposits this way in an actual app / we want to encourage them to use arb-ts.

I think we should also even consider getting rid of depositEth viaDapp and viaInbox since at this stage it might just throw devs off, but we can discuss.

Reply:
I don't think its useful to even show this here, will just confuse people.
We could instead `waitForL2()` which is the intended user flow when consuming arb-ts
here is an example https://github.com/OffchainLabs/arbitrum/blob/6ae9b968f19ff5e581fd98e935292d8442bccb15/packages/arb-ts/integration_test/eth.test.ts#L89

Reply:
We cannot actually use `waitForL2()` on this receipt here as it's an execution of a deposit function from a DApp and not the `ethBridger` class.